content/publishing-yaml/distribution.md

```diff
@@ -295,3 +295,53 @@ When the build is successful, you can see your application published to Firebase
 Project Console: https://console.firebase.google.com/project/your-project/overview
 Hosting URL: https://your-project.web.app
 ```
+
+## Publishing web applications to Netlify
+
+Publishing web applications to Netlify with Codemagic is done with the Netlify API.
+
+1. To get started with publishing web apps to Netlify with Codemagic, you will need to obtain your Netlify personal access token. In order to do that, go to user settings in Netlify and you can find personal access tokens under the Applications section.
```

Comment:
It would be good to link to Netlify docs/site on how to get this personal access token.